packages/fluxible-addons-react/package.json

```diff
@@ -15,7 +15,7 @@
     "precover": "npm run dist",
     "prepublish": "npm run dist",
     "pretest": "npm run dist",
-    "test": "../../node_modules/.bin/jest -c ../../jest.config.js",
+    "test": "../../node_modules/.bin/jest -c ../../jest.config.js packages/fluxible-addons-react/",
```

Comment:
Was this change intentional?

Reply:
Yup. If we don't pass the package folder (using the repo root as reference), jest will run all tests for all packages when we do a npm test inside the package folder.

Reply:
Forgot about this, but why didn't we have to do this with any other package?

Reply:
Actually we do, I just was a little bit lazy to add it for the other ones as well. I'll send a commit.

Reply:
Done ;)

Reply:
How are you executing the tests? If you run `lerna run test --scope fluxible-addons-react` then lerna will automatically change the cwd to that folder, does this work for you?

Reply:
If I'm working only in one package, I `cd` to it's folder and run `npm t`. But if I'm changing many packages at a time, then I run `npm t` in the root folder. So, basically, I pretend that lerna does not exist.

Your command works but it's not as easier to remember as just `npm t`, right? But if you prefer, I can undo it, no problem ;)

Reply:
Ya lets back that change out for now since its unrelated to this PR anyways. We can think about the best way to tackle that separately.

Reply:
Done! But I knew it was not a dejavu: https://github.com/yahoo/fluxible/pull/647#discussion_r486614345 :D